changelog.txt

```diff
@@ -27,13 +27,13 @@ Template for new versions:
 # Future
 
 ## New Tools
+- `devel/hello-world`: updated to show off the new Slider widget
```

Comment:
`devel/hello-world` isn't really a tool, and in the past changes to it have been logged under "Misc Improvements", so please move this to that category